Makefile

```diff
@@ -1,6 +1,7 @@
 # GLOBAL VARIABLES
 export CC=gcc
-export CFLAGS=-Wall -g -W -pedantic -std=gnu99 -lm
+export CFLAGS=-Wall -g -W -pedantic -std=gnu99
+export LDFLAGS=-lm
```

Comment:
I've separated the linker flags, because in my osx-compiler it trows some warnings:

> clang: warning: -lm: 'linker' input unused [-Wunused-command-line-argument]

But now! 👍